startupscript/setup-bashrc.sh

```diff
@@ -72,4 +72,23 @@ EOF
 
 fi
 
+if [[ "${CLOUD}" == "aws" && "${LOG_IN}" == "true" ]]; then
+
+  # Create a symlink to this workspace's AWS config file to use as the target for AWS_CONFIG_FILE.
+  readonly AWS_CONFIG_SYMLINK="${USER_WORKBENCH_CONFIG_DIR}/workspace.conf"
```

Comment:
can you add the new env var USER_WORKBENCH_CONFIG_DIR to the documentation after line 20